mozilla/addons-server
# set action_date too for already disabled decisions

Fixes: mozilla/addons#15172



### Description

Set action_date

### Testing

- Disable some content (e.g. an add-on)
- File an abuse report for it
- check reviewers 2nd level approval queue and see it empty
- (check django shell that the decision just created has an `action_date` defined)

### Checklist
